dispatch/static/manager/src/js/components/Tooltip.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+
+class Tooltip extends React.Component{
+  constructor(props) {
+    super(props)
+    
+    this.state = {
+      isTooltipDisplayed: false
+    }
+    
+    this.hideTooltip = this.hideTooltip.bind(this)
+    this.showTooltip = this.showTooltip.bind(this)
+  }
+
+  hideTooltip () {
```

Comment:
ES6 arrow functions give us a cleaner way to bind this to our react methods. 

`hideTooltip = () = > {` will automatically bind this to the context of the component and remove the need for binding in the constructor.

Reply:
Also, I am in the process of updating the code base to utilize ES6 syntax, so it will be good practice to write new components with arrow functions